sqldelight-sample/build.gradle

```diff
@@ -62,7 +57,16 @@ android {
     }
   }
 }
+
 repositories {
   mavenCentral()
+  google()
 }
 
+dependencies {
+  implementation deps.kotlin.stdlib
+  implementation deps.arch.dbFramework
+
+  implementation "com.android.support:recyclerview-v7:${versions.support}"
+  implementation "com.squareup.sqldelight:android-driver:1.0.0-alpha4"
```

Comment:
seems like we should promote this to the `versions` map as well for easy updating and so it's shared with the plugin above